cmd/registry/core/list.go

```diff
@@ -80,6 +80,35 @@ func ListAPIs(ctx context.Context,
 	return nil
 }
 
+func ListDeployments(ctx context.Context,
+	client *gapic.RegistryClient,
+	name names.Deployment,
+	filterFlag string,
+	handler DeploymentHandler) error {
+	request := &rpc.ListApiDeploymentsRequest{
+		Parent: name.Parent(),
+	}
+	filter := filterFlag
```

Comment:
Can we avoid this? Maybe just rename the argument instead of making a new variable.